torch/testing/_internal/common_utils.py

```diff
@@ -93,6 +93,18 @@
 DISABLED_TESTS_FILE = '.pytorch-disabled-tests.json'
 SLOW_TESTS_FILE = '.pytorch-slow-tests.json'
 
+disabled_tests_dict = {}
+slow_tests_dict = {}
+
+if os.getenv("IMPORT_SLOW_TESTS", ""):
+    with open(SLOW_TESTS_FILE, 'r') as fp:
+        slow_tests_dict = json.load(fp)
+        print("loaded slow tests dict")
+if os.getenv("DISABLED_TESTS_DICT", ""):
+    with open(DISABLED_TESTS_FILE, 'r') as fp:
+        disabled_tests_dict = json.load(fp)
+        print("loaded disabled tests dict")
```

Comment:
Nits(good to print size of the dict and also use warnings mechanism rather than print so that user can choose to suppress them)
```suggestion
        warnings.warn(f"loaded {len(disabled_tests_dict)} disabled tests")
```

Reply:
Shouldn't warnings be used to indicate something actually going wrong though? This feels like a very normal message.